### ISSUE TITLE
Collaboration

### ISSUE THREAD

**Issue by JamesRamm** [mmcardle/django_builder#20]
Would love to be able to share my 'session' with others in order to collaborate on designs...

**Comment by bradlishman** [comment]
Agreed. Would be great to have a user account, create a project, teams etc.
